src/Dto/TimestreamReaderDto.php

```diff
@@ -6,6 +6,10 @@
 
 final class TimestreamReaderDto extends AbstractTimestreamDto
 {
+    private ?int $maxRows = null;
+
+    private string $nextTokenToContinueReading = '';
```

Comment:
I can't seem to find where this is being used as well!

In the loop you are always calling `runQuery($params)` which will have the nextToken already available in the `parsecs`